src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+	private static final int INDEX_OF_EMPTY_STACK = -1;
+    private static final int STACK_CAPACITY = 12;
+    private static final int INDEX_OF_FULL_STACK = 11;

```

Comment:
This constant is related to STACK_CAPACITY . Changing only one leads to an error (inconsistency).